packages/core/admin/server/routes/users.js

```diff
@@ -32,6 +32,7 @@ module.exports = [
     config: {
       policies: [
         'admin::isAuthenticatedAdmin',
+        { name: 'admin::hasLicenseLimitAllowance', config: { isCreating: true } },
```

Comment:
I feel like we should first check if we have the permission to create or update a user before making license checks etc... Otherwise, we're potentially leaking the seats logic to unauthorized admin users.